### PR TITLE
fix crash with ic2 #77

### LINKED ISSUE #77: [BUG]: crash report
### Describe the bug

I don't know why the game won't start with the IC2.

### Expected behavior

Start the game normally.

### Mekanism Version

9.10.23-ALL

### Minecraft Version is this regarding?

1.7.10

### What OS are you seeing the problem on?

Windows

### Name of modpack if applicable

_No response_

### Version of said modpack if applicable

_No response_

### Screenshots

image

### The crash report in folder ./crash-reports (both server and client logs)
crash-2023-01-15_20.46.39-client.txt : [https://pastebin.com/qCr6SWw8](https://pastebin.com/qCr6SWw8)
### Please provide the following other files
Use default configuration file.

latest.txt : [https://pastebin.com/qFmSRLTV](https://pastebin.com/qFmSRLTV)

### PR DESCRIPTION
**Pull request by maggi373**

### CODE REVIEW

File: src/main/java/mekanism/common/integration/OreDictManager.java [hunk 1]
```diff
@@ -66,9 +66,14 @@ public static void init() {
 			RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), new ItemStack(MekanismItems.OtherDust, 4, 1));
 		}
 
-		if (OreDictionary.getOres("itemRubber").size() > 0) {
+		if (OreDictionary.getOres("itemRawRubber").size() > 0) {
```
Comment (DrParadox7) [inline src/main/java/mekanism/common/integration/OreDictManager.java:69]: Why not use `!OreDictionary.getOres("itemRawRubber").isEmpty()` instead?

Comment (maggi373) [reply src/main/java/mekanism/common/integration/OreDictManager.java:69]: well i'm lazy and didnt bothered to change that?, does the same anyways.

Comment (maggi373) [reply src/main/java/mekanism/common/integration/OreDictManager.java:69]: feel free to change it, remember to test it aswell

**Comment by DrParadox7** [comment]
Could you also set `osmiumcompat` to final and capitalize the "C" in compat in the code below:

```
private static final List<String> minorCompat = Arrays.asList("Nickel", "Aluminum");
private static final List<String> siliconcompat = Arrays.asList("itemSilicon", "silicon");
private static List<String> osmiumcompat = new ArrayList<>();
```
https://github.com/Thorfusion/Mekanism-1.7.10-Community-Edition/blob/ab77661778d0ed0af4a1ef99122070dca88f83d2/src/main/java/mekanism/common/integration/OreDictManager.java#L39

**Comment by maggi373** [comment]
> 

its not an oredict

File: src/main/java/mekanism/common/integration/OreDictManager.java (hunk 1 above)
Comment (DrParadox7) [reply src/main/java/mekanism/common/integration/OreDictManager.java:69]: Alright, I'll eventually clean up the code in future commits

**Review by DrParadox7 — APPROVED** [review]

**Comment by maggi373** [comment]
> Could you also set `osmiumcompat` to final and capitalize the "C" in compat in the code below:
> 
> ```
> private static final List<String> minorCompat = Arrays.asList("Nickel", "Aluminum");
> private static final List<String> siliconcompat = Arrays.asList("itemSilicon", "silicon");
> private static List<String> osmiumcompat = new ArrayList<>();
> ```
> 
> https://github.com/Thorfusion/Mekanism-1.7.10-Community-Edition/blob/ab77661778d0ed0af4a1ef99122070dca88f83d2/src/main/java/mekanism/common/integration/OreDictManager.java#L39

also final on osmiumcompat wont work due to that strings are added later in the code

**Pull request merged**